packages/discovery-provider/src/queries/get_top_playlists_es.py

```diff
@@ -12,7 +11,7 @@
 
 def get_top_playlists_es(kind, args):
     esclient = get_esclient()
-    current_user_id = get_current_user_id(required=False)
+    current_user_id = args.get("current_user_id")
```

Comment:
https://github.com/AudiusProject/audius-protocol/blob/808456542f5f9b4b587371363141d265cb3b17e2/packages/discovery-provider/src/api/v1/playlists.py#L440

https://github.com/AudiusProject/audius-protocol/blob/808456542f5f9b4b587371363141d265cb3b17e2/packages/discovery-provider/src/queries/queries.py#L481

for my own sake, looking at the callers, they are both doing a user_id => current_user_id transformation

Reply:
https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/src/queries/query_helpers.py#L72-L73 is what was being used actually, and the caller of `get_top_playlists_es`, which is https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/src/api/v1/playlists.py#L440 is the one that uses the correct version of `get_current_user_id`